unittests/testLocalResourceRetriever.cpp

```diff
@@ -38,78 +38,91 @@
 #include "dart/common/LocalResourceRetriever.h"
 #include "TestHelpers.h"
 
+using dart::common::Uri;
 using dart::common::Resource;
 using dart::common::LocalResourceRetriever;
 
 TEST(LocalResourceRetriever, exists_UnsupportedUri_ReturnsFalse)
 {
   LocalResourceRetriever retriever;
-  EXPECT_FALSE(retriever.exists("unknown://test"));
+  EXPECT_FALSE(retriever.exists(Uri("unknown://test")));
 }
 
 TEST(LocalResourceRetriever, exists_FileUriDoesNotExist_ReturnsFalse)
 {
   LocalResourceRetriever retriever;
-  EXPECT_FALSE(retriever.exists(DART_DATA_PATH"does/not/exist"));
+  EXPECT_FALSE(retriever.exists(Uri::createFromPath(DART_DATA_PATH"does/not/exist")));
 }
 
 TEST(LocalResourceRetriever, exists_PathDoesNotExist_ReturnsFalse)
 {
   LocalResourceRetriever retriever;
-  EXPECT_FALSE(retriever.exists(DART_DATA_PATH"does/not/exist"));
+  EXPECT_FALSE(retriever.exists(Uri::createFromPath(DART_DATA_PATH"does/not/exist")));
 }
 
 TEST(LocalResourceRetriever, exists_FileUriDoesExists_ReturnsTrue)
 {
   LocalResourceRetriever retriever;
-  EXPECT_TRUE(retriever.exists("file://" DART_DATA_PATH "skel/cube.skel"));
+#ifdef _WIN32
+  EXPECT_TRUE(retriever.exists(Uri::createFromPath("file:///" DART_DATA_PATH "skel/cube.skel")));
+#else
+  EXPECT_TRUE(retriever.exists(Uri::createFromPath("file://" DART_DATA_PATH "skel/cube.skel")));
+#endif
```

Comment:
Then this test would be the same with `exists_FileUriDoesNotExists_ReturnsTrue` (this name should be changed to `exists_PathDoesExists_ReturnsTrue`).

I think this test is checking if file URIs work so we might need to use `Uri::createFromString` instead of removing `file://`.